mid-assessment/q7.rb

```diff
@@ -0,0 +1,14 @@
+# Implement a program to check if a given string is a palindrome or not.
+def pallindrome?
+  print 'Enter a string: '
+  pall = gets.chomp
```

Comment:
Use meaningful variables. like here variable pall is not making the sense for what purpose you are using it. instead you can use pallindrome or given_string etc.

Reply:
ok